scratchwork/main.go

```diff
@@ -33,13 +34,48 @@ var (
 	// todo udpate the legendHost based on the legendsPort (when it changes)
 	// url.Parse -- but I personally find this code hard to read.
 	legendHost = fmt.Sprintf("http://localhost:%d", legendsPort)
+
+	cardCodeToName = make(map[string]string)
 )
 
+type PositionalRectangles struct {
+	GameState   string
+	Rectangles  Rectangles
+}
+
+type Rectangles []struct {
+	CardCode    string
+	LocalPlayer bool
+}
+
 type StaticDeckList struct {
 	DeckCode    string
 	CardsInDeck map[string]int
 }
 
+type Cards []struct {
+	CardCode    string
+	CardName    string `json:"name"`
+}
+
+func fillCardCodeToNameMap() error{
+	jsonFile, err := os.Open("../en_us/data/set1-en_us.json")
```

Comment:
eventually, we should download this from the website since the cards will change rapidly. For now, this is fine

Reply:
Add a todo comment for the above, but just wanted to note this for the future on why we left this here.

Reply:
Use this to get the code to GET from website:
https://mholt.github.io/curl-to-go/

Unzip in golang
https://stackoverflow.com/questions/20357223/easy-way-to-unzip-file-with-golang